### DISCUSSION TITLE
Packaging

### DISCUSSION THREAD

**Discussion by jaidetree** [jank-lang/jank#16] [Issue Triage] [answered]
Any plans for packaging and package managers? Just interested in what people envision for this project.

**Comment by jeaye** [comment, marked as answer]
My plan, out of the gate, is to hook into leiningen and use the same dep management system as Clojure. Lein is powerful and well-known, so if it can work, I think it's the top choice until jank is big enough to need something else. Performance will likely be the biggest concern. Even then, I would probably just try to compile lein with jank so that it starts up faster.

**Reply by jaidetree** [reply]
Ooh I like that idea and if the compiled version of lein was included that might be really nice to get moving quickly with it like js.
